--- aai.egi.eu.php.orig	2023-11-21 10:49:53.692258927 +0100
+++ aai.egi.eu.php	2023-11-21 10:52:31.549282064 +0100
@@ -167,4 +167,24 @@
     array (
     ),
   ),
+  'tags' =>
+  array (
+    0 => 'er_infras',
+  ),
+  'authproc' =>
+  array(
+    10 => array(
+       'class' => 'authorize:Authorize',
+       'urn:oid:1.3.6.1.4.1.5923.1.1.1.7' =>
+       array(
+           '@^urn:mace:egi.eu:res:rcauth#.*$@'
+       ),
+    ),
+    # copy voPersonID into eduPersonUniqueId for legacy EGI Check-in users that
+    # already have a RCauth DN.
+    60 => array(
+       'class' => 'core:AttributeCopy',
+       'urn:oid:1.3.6.1.4.1.25178.4.1.6' => 'urn:oid:1.3.6.1.4.1.5923.1.1.1.13',
+    ),
+  ),
 );
