--- wayf.rcauth.eu.php.orig	2019-11-28 10:06:22.506400631 +0100
+++ wayf.rcauth.eu.php	2019-11-27 13:24:58.511731483 +0100
@@ -50,4 +50,21 @@
       'givenName' => 'Operator',
     ),
   ),
+  'tags' =>
+  array (
+    0 => 'preferred',
+  ),
+  'UIInfo' => 
+  array (
+    'DisplayName' => 
+    array (
+      'en' => 'RCauth WAYF',
+      'nl' => 'RCauth WAYF',
+    ),
+  ),
+  'OrganizationDisplayName' =>
+  array (
+    'en' => 'RCauth WAYF',
+    'nl' => 'RCauth WAYF',
+  ),
 );
