Description: comparison with expiry timestamp is wrongly ordered
Author: David Groep <davidg@nikhef.nl>
--- a/modules/core/templates/frontpage_federation.twig	2019-11-06 14:29:31.000000000 +0100
+++ b/modules/core/templates/frontpage_federation.twig	2019-11-21 14:47:41.000000000 +0100
@@ -46,7 +46,7 @@
             {% endif -%}</a>
 
             {%- if entity.expire is defined %}
-            {% if entity.expire > date().timestamp %}
+            {% if entity.expire < date().timestamp %}
             <span class="entity-expired"> (expired {{ ((date().timestamp - entity.expire) / 3600) }} hours ago)</span>
             {% else %} (expires in {{ ((entity.expire - date().timestamp) / 3600) }} hours){% endif -%}{% endif %}
             </li>
