Description: add our tags to eduGAIN metadata
Author: Mischa Sallé <msalle@nikhef.nl>
Author: Tamas Balogh <tamasb@nikhef.nl>
--- a/modules/metarefresh/lib/MetaLoader.php	2019-11-26 14:18:26.000000000 +0100
+++ b/modules/metarefresh/lib/MetaLoader.php	2019-11-27 12:38:15.000000000 +0100
@@ -206,6 +206,30 @@
             $template = null;
             if (array_key_exists('template', $source)) {
                 $template = $source['template'];
+
+                if(isset($source['tagmapping']) && !empty($source['tagmapping'])) {
+                    $tagmapping = $source['tagmapping'];
+
+                    $idpmetadata = $entity->getMetadata20IdP();
+                    if(array_key_exists('RegistrationInfo', $idpmetadata)) {
+                        $regAuthInfo = $idpmetadata['RegistrationInfo'];
+                        if(array_key_exists('registrationAuthority', $regAuthInfo)) {
+                            $regAuth = $regAuthInfo['registrationAuthority'];
+                            Logger::debug('Found the Registration Auth ' . $regAuth . " for ". $entity->getEntityId());
+                            if (array_key_exists($regAuth,$tagmapping)) {
+                                array_push($template['tags'],$tagmapping[$regAuth]);
+                            } elseif (array_key_exists('*',$tagmapping)) {
+                                array_push($template['tags'],$tagmapping['*']);
+                            } else {
+                                Logger::info('No tag map definition for ' . $regAuth . ' (entityID '. $entity->getEntityId() . ")\n");
+                            }
+                        } else {
+                            Logger::info('No Registration Authority for '. $entity->getEntityId() . "\n");
+                        }
+                    } else {
+                        Logger::info('No Registration Info for ' . $entity->getEntityId() . "\n");
+                    }
+                }
             }
 
             if (in_array('shib13-sp-remote', $this->types, true)) {
